server/plugin/plugin.go

```diff
@@ -982,13 +981,19 @@ func (p *Plugin) isUserOrganizationMember(githubClient *github.Client, user *git
 		return false
 	}
 
-	isMember, _, err := githubClient.Organizations.IsMember(context.Background(), organization, *user.Login)
-	if err != nil {
-		p.client.Log.Warn("Failled to check if user is org member", "GitHub username", *user.Login, "error", err.Error())
-		return false
+	orgList, _ := getOrganizations(organization)
+	isMemberOfAllOrgs := true
+	for _, org := range orgList {
+		isMember, _, err := githubClient.Organizations.IsMember(context.Background(), org, *user.Login)
+		if err != nil {
+			p.client.Log.Warn("Failled to check if user is org member", "GitHub username", *user.Login, "error", err.Error())
+			return false
+		}
+
+		isMemberOfAllOrgs = isMemberOfAllOrgs && isMember
```

Comment:
Is that really the expected logic? I would expect the exclude org member flag to apply for every use that is part of one org, not all of them.